doc/classes/RichTextEffect.xml

```diff
@@ -6,10 +6,16 @@
 	<description>
 		A custom effect for use with [RichTextLabel].
 		[b]Note:[/b] For a [RichTextEffect] to be usable, a BBCode tag must be defined as a member variable called [code]bbcode[/code] in the script.
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		# The RichTextEffect will be usable like this: `[example]Some text[/example]`
 		var bbcode = "example"
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		// The RichTextEffect will be usable like this: `[example]Some text[/example]`
+		public string bbcode = "example"; // Bbcode?
```

Comment:
```suggestion
		public string bbcode = "example";
```

Reply:
I added the comment, because I am not sure wheter or not `bbcode`should be capitalized according to the style guide.

Reply:
Oh, I see. I think it's good as is.